addons/binding/org.openhab.binding.knx/build.properties

```diff
@@ -0,0 +1,12 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/calimero-core-2.3.jar,\
+               lib/calimero-device-2.3.jar,\
+               lib/calimero-rxtx-2.3.jar,\
+               about.html,\
+               README.md
```

Comment:
Why did you include the README in the binary?

Reply:
did I? 😉